plugins/superio/fu-plugin-superio.c

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2018 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include "fu-plugin.h"
+#include "fu-plugin-vfuncs.h"
+
+#include "fu-superio-device.h"
+
+gboolean
+fu_plugin_coldplug (FuPlugin *plugin, GError **error)
+{
+	g_autoptr(FuSuperioDevice) dev = NULL;
+	g_autoptr(FuDeviceLocker) locker = NULL;
+
+	/* create device and unlock */
+	dev = fu_superio_device_new ("/dev/port");
+	locker = fu_device_locker_new_full (dev,
+					    (FuDeviceLockerFunc) fu_superio_device_open,
+					    (FuDeviceLockerFunc) fu_superio_device_close,
+					    error);
+	if (locker == NULL)
+		return FALSE;
+	if (!fu_superio_device_probe (dev, error))
```

Comment:
I'm fairly wary of randomly probing at an EC unless you know you should.  Could this be quirked to only run on SMBIOS strings that correspond to manufacturers who solely use this chip (or chips that are known to respond properly)?

Reply:
Fixed.